packages/redux/src/users/authentication/__tests__/reducer.test.ts

```diff
@@ -121,6 +123,129 @@ describe('authentication reducer', () => {
+      actionTypes.FETCH_USER_EXTERNAL_LOGINS_REQUEST,
+      actionTypes.REMOVE_USER_EXTERNAL_LOGIN_REQUEST,
+    ])('should handle %s action type', actionType => {
+      const state: AuthenticationState = {
+        ...fromReducer.INITIAL_STATE,
+        externalLogins: {
+          result: null,
+          error: toBlackoutError(new Error('dummy error')),
+          isLoading: false,
+        },
+      };
+
+      const externalLogins = {
+        result: null,
+        error: null,
+        isLoading: true,
+      };
+
+      expect(
+        reducer(state, {
+          type: actionType,
+        }).externalLogins,
+      ).toEqual(externalLogins);
+    });
+
+    it.each([
+      actionTypes.FETCH_USER_EXTERNAL_LOGINS_FAILURE,
+      actionTypes.REMOVE_USER_EXTERNAL_LOGIN_FAILURE,
+    ])('should handle %s action type', actionType => {
+      const state: AuthenticationState = {
+        ...fromReducer.INITIAL_STATE,
+        externalLogins: {
+          result: null,
+          error: null,
+          isLoading: true,
+        },
+      };
+
+      const externalLogins = {
+        result: null,
+        error: toBlackoutError(new Error('dummy error')),
+        isLoading: false,
+      };
+
+      expect(
+        reducer(state, {
+          type: actionType,
+          payload: { error: toBlackoutError(new Error('dummy error')) },
+        }).externalLogins,
+      ).toEqual(externalLogins);
+    });
+
+    it('should handle FETCH_USER_EXTERNAL_LOGINS_SUCCESS action type', () => {
+      const state: AuthenticationState = {
+        ...fromReducer.INITIAL_STATE,
+        externalLogins: {
+          result: null,
+          error: null,
+          isLoading: true,
+        },
+      };
+
+      const expectedResult = expectedUserExternalLoginsPayload;
+
+      expect(
+        reducer(state, {
+          type: actionTypes.FETCH_USER_EXTERNAL_LOGINS_SUCCESS,
+          payload: expectedResult,
+        }).externalLogins,
+      ).toEqual({
+        error: null,
+        isLoading: false,
+        result: expectedResult,
+      });
+    });
+
+    it('should handle REMOVE_USER_EXTERNAL_LOGIN_SUCCESS action type', () => {
+      const state: AuthenticationState = {
+        ...fromReducer.INITIAL_STATE,
+        externalLogins: {
+          result: null,
+          error: null,
+          isLoading: true,
+        },
+      };
+
+      const expectedResult = expectedUserExternalLoginsPayload;
+
+      expect(
+        reducer(state, {
+          type: actionTypes.REMOVE_USER_EXTERNAL_LOGIN_SUCCESS,
+          payload: expectedResult,
+        }).externalLogins,
+      ).toEqual({
+        error: null,
+        isLoading: false,
+        result: [],
+      });
+    });
+
+    it('should handle other actions by returning the previous state', () => {
+      const state = {
+        externalLogins: {
+          result: null,
+          error: 'error',
+          isLoading: false,
+        },
+      } as unknown as AuthenticationState;
```

Comment:
Avoid having this please, you can easily have the state complying with AuthenticationState by using the fromReducer and having the `error` property be a `BlackoutError` instance

Reply:
Sorry, missed it. Fixed.